src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -42,9 +41,39 @@ public function getAccount() {
    * Set the current object account.
    *
    * @param AccountInterface $account
+   *   The user object.
+   * @return $this
    */
-  public function setAccount(AccountInterface $account) {
+  public function setAccount(AccountInterface $account = NULL) {
     $this->currentUser = $account;
+    return $this;
+  }
+
+  /**
+   * Get the field configuration.
+   *
+   * @param $key
+   *   Specific key from the configuration. Optional.
+   *
+   * @return array|string
+   *   Value of a configuration or all the configurations.
+   */
+  public function getConfiguration($key = NULL) {
+
+    if (!$key) {
```

Comment:
For me it just makes for code that is less prone to bugs. `!$key` is just
any falsey value, which is not necessarily NULL.
On 8 Nov 2015 14:18, "Roy Segall" <notifications@github.com> wrote:

> In src/Plugin/EntityReferenceSelection/OgSelection.php
> <https://github.com/amitaibu/og/pull/11#discussion_r44225136>:
>
> >      $this->currentUser = $account;
> > +    return $this;
> > +  }
> > +
> > +  /**
> > +   * Get the field configuration.
> > +   *
> > +   * @param $key
> > +   *   Specific key from the configuration. Optional.
> > +   *
> > +   * @return array|string
> > +   *   Value of a configuration or all the configurations.
> > +   */
> > +  public function getConfiguration($key = NULL) {
> > +
> > +    if (!$key) {
>
> Don't think we need this. By default the key is NULL and if no value was
> passed !$key will return TRUE and the method will return all the
> configurations.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/11/files#r44225136>.
>